apps/frontend/src/components/global/checklist/ChecklistRuleInfoBody.vue

```diff
@@ -0,0 +1,61 @@
+<template>
```

Comment:
additionally for the severity override stuff, have like a pencil button or something show up so that it's possible to edit the text without changing the severity category again (i.e. you like the category you overrided it to, but you just wanna adjust the justifaction).  clicking the button should make the sheet pop up same as if you were trying to change the severity category.  same rule applies where you need to have at least some text - can't edit it to become the empty string.